data-pipeline/src/trace_exporter/mod.rs

```diff
@@ -1448,7 +1347,12 @@ mod tests {
         datagram.trim_matches(char::from(0)).to_string()
     }
 
-    fn build_test_exporter(url: String, dogstatsd_url: String) -> TraceExporter {
+    fn build_test_exporter(
```

Comment:
Can you use this helper function instead of creating exporters in the [new test](https://github.com/DataDog/libdatadog/pull/898/files#diff-ceb25b8be20e4522d23de0732e538c8c49257277a37edbc8bd30af04de55f4ecR1713) you added?